src/otx/core/model/anomaly.py

```diff
@@ -221,13 +223,24 @@ def configure_optimizers(self) -> tuple[list[torch.optim.Optimizer], list[torch.
             return optimizer(params=params)
         return super().configure_optimizers()  # type: ignore[misc]
 
+    def validation_step(
+        self,
+        inputs: AnomalyModelInputs,
+        batch_idx: int = 0,
+    ) -> STEP_OUTPUT:
+        """Call validation step of the anomalib model."""
+        raise NotImplementedError
+
```

Comment:
It's just a question. Is it necessary?

Reply:
Final anomaly model is inherited from several classes including this one, and to workaround missing forward implementation I need to call `validation_step` from the other anomaly-related class here. That's why I need this declaration. This class is a subject of the future anomalib integration refactoring planned by Ashwin and Dick.